src/server/extension-context-builder.ts

```diff
@@ -0,0 +1,13 @@
+import { GraphQLExtension } from 'graphql-extensions';
+
+export class ExtensionContextBuilder extends GraphQLExtension {
+    willSendResponse(responseContext: any) {
+        const { context, graphqlResponse } = responseContext;
+
+        if (context.headers.dataVersion !== undefined) {
```

Comment:
what happens if context.headers is undefined?

Reply:
its empty (never undefined)